src/slider/index.html

```diff
@@ -32,8 +32,8 @@
 </head>
 <body>
 	<main>
-		<!--Slider that slides horizontally-->
-		<tp-slider flexible-height="yes" infinite="yes" swipe="yes">
+		<!--Slider that slides horizontally with responsive settings.-->
+		<tp-slider flexible-height="yes" infinite="yes" swipe="yes" responsive="[{&quot;media&quot;:&quot;(max-width: 600px)&quot;,&quot;flexible-height&quot;:&quot;yes&quot;,&quot;infinite&quot;:&quot;no&quot;,&quot;swipe&quot;:&quot;yes&quot;,&quot;auto-slide-interval&quot;:2000,&quot;per-view&quot;:1,&quot;step&quot;:2},{&quot;media&quot;:&quot;(max-width: 300px)&quot;,&quot;flexible-height&quot;:&quot;yes&quot;,&quot;infinite&quot;:&quot;yes&quot;,&quot;swipe&quot;:&quot;yes&quot;,&quot;behaviour&quot;:&quot;fade&quot;,&quot;auto-slide-interval&quot;:2000,&quot;per-view&quot;:2,&quot;step&quot;:1}]">
```

Comment:
@imranhsayed Can we make this more readable by doing: `responsive='[{"media": "(max-width..." ...`